src/native/minipal/cpufeatures.c

```diff
@@ -530,6 +545,113 @@ int minipal_getcpufeatures(void)
 
 #endif // HOST_ARM64
 
+#if defined(HOST_RISCV64)
+
+#if defined(HOST_UNIX)
+
+#if HAVE_HWPROBE_H
+
+    struct riscv_hwprobe pairs[1] = {
+        {RISCV_HWPROBE_KEY_IMA_EXT_0, 0}
+    };
+
```

Comment:
nit: move `long sys_riscv_hwprobe` after this definition (so everything is under `HOST_RISCV64`).